modules/hello-world/pages/start-using-ottoman.adoc

```diff
@@ -109,8 +116,40 @@ async function main() {
 }
 ----
 
-Now, let's create a connection to our Couchbase Server.
+Now, let's create a connection to our database.
+
+[{tabs}]
+====
+Couchbase Capella::
++
+--
+To connect to Couchbase Capella, be sure to set up credentials for your database by xref:cloud:get-started:cluster-and-data.adoc#credentials[following these steps].
+Capella requires mandatory use of TLS (Transport Layer Security), therefore a security certificate must be provided when connecting.
 
+If you don't already have your security certificate, you can download it from the Capella UI:
+
+. Go to the "Clusters" section, in the left hand navigation.
+. Click on the relevant cluster.
+. Select the "Connect" tab and scroll down to the "Security Certificate" section.
+. Download the "Root Certificate".
+
+[source,javascript]
+----
+include::hello-world:example$start-using-ottoman-cloud.js[tag=connect]
+----
+
+TIP: If your application is running in a xref:ref:client-settings.adoc#constrained-network-environments[constrained network environment] (i.e., your laptop), you may experience timeout issues.
+To mitigate this, you can tweak your connection settings to account for any latency or bandwidth issues that may arise.
+
+Couchbase Capella uses xref:cloud:organizations:organization-projects-overview.adoc[Roles] to control user access to database resources.
+For the purposes of this example, we can use the _Organization Owner_ role automatically assigned to a user account during installation of the Capella cluster.
+This role grants full access to a Capella organization, including full data access to all projects and clusters.
+In a production scenario, we strongly recommend setting up users with more granular access roles as a best practice.
```

Comment:
Worth putting _strongly recommend_ in italics.or possibly some JS to get it to jump off the screen and tap people on the shoulder (less violent than slapping them round the face, but the minimum to make sure everyone gets the point ;) )

Reply:
Hahaha! Brings a whole new meaning to interactive docs 🤣